hazelcast/src/main/java/com/hazelcast/core/server/StartServer.java

```diff
@@ -33,6 +37,16 @@ private StartServer() {
      * @param args none
      */
     public static void main(String[] args) {
-        Hazelcast.newHazelcastInstance(null);
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance(null);
+        if (System.getProperty("member.tag") != null) {
+            PrintWriter printWriter = null;
+            try {
+                printWriter = new PrintWriter("ports/" + System.getProperty("member.tag"));
+            } catch (FileNotFoundException e) {
+                e.printStackTrace();
```

Comment:
Bleh.. e.printStackTrace. 

Do you want to continue if the file is not found? If yes; then make a log.warning message. If you don't want to continue, just propagate the exception and let the main fail with an exception.